packages/flutter/lib/src/gestures/drag_details.dart

```diff
@@ -185,21 +192,36 @@ class DragUpdateDetails {
   ///    coordinate space of the event receiver.
   final Offset globalPosition;
 
+  /// The kind of the device that initiated the event.
+  final PointerDeviceKind? kind;
+
   /// The local position in the coordinate system of the event receiver at
   /// which the pointer contacted the screen.
   ///
   /// Defaults to [globalPosition] if not specified in the constructor.
   final Offset localPosition;
 
+  /// A delta offset from the point where the drag initially contacted
+  /// the screen to the point where the pointer is currently located (the
+  /// present [globalPosition]) when this callback is triggered.
+  final Offset offsetFromOrigin;
+
+  /// A local delta offset from the point where the drag initially contacted
```

Comment:
maybe
```
A delta offset from the point where the drag initially contacted the screen to the point where the pointer is currently located in local coordinates relative to <TextField?>
```

Reply:
I don't think we should add the TextField specification since these Details object is used outside of TextField.

Reply:
That is just place holder. When it say local coordinates, it only makes sense if it mention what it is relative to.